cmd/grpc/main.go

```diff
@@ -46,7 +46,7 @@ func main() {
 	}
 	defer macaroonService.Close()
 
-	var pass = []byte("hello") //TODO this should be passed as flag/env?
+	var pass = []byte(config.GetString(config.MacaroonRootKeyPass))
```

Comment:
I would move this in the Unlock RPC, so we can re-use that wallet password also for macaroons. Doable?